ext/civi_case/civi_case.php

```diff
@@ -17,3 +17,22 @@ function civi_case_civicrm_managed(&$entities, $modules) {
     );
   }
 }
+
+/**
+ * Applies Case permissions to Activities
+ *
+ * @implements CRM_Utils_Hook::selectWhereClause
+ */
+function civi_case_civicrm_selectWhereClause($entityName, &$clauses, $userId, $conditions) {
```

Comment:
Just reported a regression for this change and specifically this function, see https://lab.civicrm.org/dev/core/-/issues/4769